Screenbox/Extensions/ListViewExtensions.cs

```diff
@@ -0,0 +1,271 @@
+    public static CornerRadius GetItemCornerRadius(ListViewBase element)
+    {
+        return (CornerRadius)element.GetValue(ItemCornerRadiusProperty);
+    }
+
+    /// <summary>
+    /// Sets the value of the ItemCornerRadius attached property on the specified <see cref="ListViewBase"/>.
+    /// </summary>
+    /// <param name="element">The <see cref="ListViewBase"/> on which to set the property value.</param>
+    /// <param name="value">The new value to set the property to.</param>
+    public static void SetItemCornerRadius(ListViewBase element, CornerRadius value)
+    {
+        element.SetValue(ItemCornerRadiusProperty, value);
+    }
+
+    /// <summary>
+    /// Identifies the attached dependency property that specifies the margin of the <see cref="ListViewBase"/> item.
+    /// </summary>
+    public static readonly DependencyProperty ItemMarginProperty = DependencyProperty.RegisterAttached(
+        "ItemMargin", typeof(Thickness), typeof(ListViewExtensions), new PropertyMetadata(null, OnItemMarginPropertyChanged));
+
+    /// <summary>
+    /// Gets the value of the ItemMargin attached property from the specified <see cref="ListViewBase"/>.
+    /// </summary>
+    /// <param name="element">The <see cref="ListViewBase"/> from which the property value is read.</param>
+    /// <returns>The current value of the ItemMargin attached property on the specified <see cref="ListViewBase"/>.</returns>
+    public static Thickness GetItemMargin(ListViewBase element)
+    {
+        return (Thickness)element.GetValue(ItemMarginProperty);
+    }
+
+    /// <summary>
+    /// Sets the value of the ItemMargin attached property on the specified <see cref="ListViewBase"/>.
+    /// </summary>
+    /// <param name="element">The <see cref="ListViewBase"/> on which to set the property value.</param>
+    /// <param name="value">The new value to set the property to.</param>
+    public static void SetItemMargin(ListViewBase element, Thickness value)
+    {
+        element.SetValue(ItemMarginProperty, value);
+    }
+
+    /// <summary>
+    /// Identifies the attached dependency property that specifies the minimum height of the <see cref="ListView"/> item.
+    /// </summary>
+    public static readonly DependencyProperty ItemMinHeightProperty = DependencyProperty.RegisterAttached(
+        "ItemMinHeight", typeof(double), typeof(ListViewExtensions), new PropertyMetadata(null, OnItemMinHeightPropertyChanged));
+
+    /// <summary>
+    /// Gets the value of the ItemMinHeight attached property from the specified <see cref="ListViewBase"/>.
+    /// </summary>
+    /// <param name="element">The <see cref="ListViewBase"/> from which the property value is read.</param>
+    /// <returns>The current value of the ItemMinHeight attached property on the specified <see cref="ListViewBase"/>.</returns>
+    public static double GetItemMinHeight(ListViewBase element)
+    {
+        return (double)element.GetValue(ItemMinHeightProperty);
```

Comment:
Nitpick: I see the default value for `ItemMinHeight` property is null. But here we are doing an unsafe cast. If that property is ever be null, an exception will be thrown here.

Reply:
Oops I missed this reply. Generally, if the property has a value type (like a `double`), then the default value should be the default value of the type (e.g. `0d`). Unless there is an explicit need for a nullable value type, the property should not be initialized as `null`. If you need to null-check it for something, then you should declare the type as nullable (like `double?`). Value type and `null` should not mix.